exporter/metric/metric_test.go

```diff
@@ -73,6 +88,16 @@ func TestRecordToMpb(t *testing.T) {
 	cps := test.NewCheckpointSet(res)
 	ctx := context.Background()
 
+	cloudMock := cloudmock.NewCloudMock()
```

Comment:
Similarly, this test only invokes the `recordToMpb` and doesn't actually export anything, so adding mock is redundant.